docs/faq.md

```diff
@@ -2,26 +2,47 @@
 
 ## What is LinkStack?
 
-LinkStack is a modification of LinkStack, an open source project with the purpose of enabling people to host their own alternatives to services like Linktree and many.link.
+LinkStack is a modification of LittleLink Admin, an open source project with the purpose of enabling people to host their own alternatives to services like Linktree and many.link.
+
+Until April 2023, the LinkStack project was known as "LittleLink Custom."
 
 
 ## Why does LinkStack exist?
 
-This project is envisioned to be a drag and drop alternative to other solutions. No coding or command line setup required; just plug and play.
+This project aims to be a drag and drop alternative to other solutions, making it more accessible and ultimately easier to use for first-time users to set up and use it. No coding or command line setup required; just plug and play on a server (that meets the requirements) of your choice.
 
 
 ## What can LinkStack do for me?
 
 Many social media platforms only allow you to add one link, with this you can simply link to your LinkStack page and have all the links you want displayed on one site. You can share all your links to your social media platform or 
-important links to easy accessible and hosted on your own web-server or web-hosting provider. On this website, other users can register and create their own links, you can access other user via the Admin Panel.
+important links to easy accessible and hosted on your own web-server or web-hosting provider. On this website, other users can register and create their own links, you can access other users via the Admin Panel.
 
 
 ## What do I need to use LinkStack?
 
-To host your own instance of LinkStack, you will need a web server running at least PHP 8.0.
+To host your own instance of LinkStack, you will need a web server running at least PHP 8.1

```

Comment:
The full stop is missing here

Reply:
still missing ^^

Reply:
I added and changed it back to 8.0 (per a conversation with the main maintainer last month.)

I also linked out to the newly proposed access admin page.